### PR TITLE
First

### PR DESCRIPTION
**Pull request by symm**
Hi! 👋

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
```
Comment (symm) [inline .idea/misc.xml]: XML is awful

File: .idea/ProjectChicken.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
```
Comment (symm) [inline .idea/ProjectChicken.iml]: lol

Comment (symm) [reply .idea/ProjectChicken.iml]: test

Comment (symm) [reply .idea/ProjectChicken.iml]: test

Comment (symm) [reply .idea/ProjectChicken.iml]: test

Comment (symm) [reply .idea/ProjectChicken.iml]: hello derek

File: .idea/misc.xml (hunk 1 above)
Comment (dezzak) [reply .idea/misc.xml]: 👍

**Pull request closed without merging**